Modules/Tests/StorageTests/CoreData/CoreDataIterativeMigratorTests.swift

```diff
@@ -187,12 +187,12 @@ final class CoreDataIterativeMigratorTests: XCTestCase {
         // Start another container. If there is no error, then the migration was successful.
         let _ = try startPersistentContainer(storeURL: storeURL, storeType: storeType, model: targetModel)
 
-        // There are 9 destroyed URLs because 9 migration steps should have happened between
-        // "Model" to "Model 10":
+        // There are 10 destroyed URLs because 9 migration steps should have happened between
+        // "Model 30" to "Model 40":
         //
-        // 1 → 2, 2 → 3, 3 → 4, 4 → 5, 5 → 6, 6 → 7, 7 → 8, 8 → 9, 9 → 10
+        // 30 → 31 → ... → 40
         //
-        XCTAssertEqual(spyCoordinator.destroyedURLs.count, 9)
+        XCTAssertEqual(spyCoordinator.destroyedURLs.count, 10)
```

Comment:
There was no Model 1. It went from Model to Model 2 hence why I changed the number of steps to assert to 10